tests/test_points.py

```diff
@@ -84,3 +84,78 @@ def test_NoToken(self):
     def test_InvalidToken(self):
         response = self.get(self.VALID_URL, self.bogus_token())
         self.assert_invalid_token(response)
+
+
+class TestPointsPost(EndpointTestCase):
+    """
+    Test for '/api/students/<int:student_id>/points/' POST endpoint.
+    """
+
+    # Fixture specific URL to available student data.
+    VALID_URL = "/api/students/2/points/"
+    # Fixture specific URL to data not available for current user.
+    NOT_PERMITTED_URL = "/api/students/1/points/"
+    # Fixture specific URL to invalid student ID.
+    NOT_FOUND_URL = "/api/students/12345/points/"
+    # Fixture specific valid data to post a prize.
+    VALID_POST_PRIZE = {"content_type": "prize", "object_id": "2"}
+    # Fixture specific valid data to post a task.
+    VALID_POST_TASK = {"content_type": "task", "object_id": "2"}
+    # Fixture specific data to post an invalid content type.
+    INVALID_POST_TYPE = {"content_type": "asdf", "object_id": "1"}
+
+    def test_Prize_Success(self):
+        student_points = self.get_student_points(2)
+        response = self.post(self.VALID_URL, self.VALID_POST_PRIZE)
+        # General assertions.
+        assert response.status_code == status.HTTP_201_CREATED
+        # Fixture specific assertions.
+        response_json = response.json()
+        student_prize = self.get("/api/students/2/prize/2/").json()
+        assert response_json["value"] == int(student_prize["value"])
+        assert response_json["student"] == int(student_prize["student"])
+        assert response_json["object_id"] == student_prize["pk"]
+        assert response_json["assigner"] == 1
+        assert response_json["points_type"] == "prize"
+        assert response_json["content_type"] == 11
+
+        points_spent = self.get_student_points(2)
+        assert student_points - response_json["value"] == points_spent
+
+    def test_Task_Success(self):
+        student_points = self.get_student_points(2)
+        response = self.post(self.VALID_URL, self.VALID_POST_TASK)
+        # General assertions.
+        assert response.status_code == status.HTTP_201_CREATED
+        # Fixture specific assertions.
+        response_json = response.json()
+        student_task = self.get("/api/students/2/task/2/").json()
+        assert response_json["value"] == int(student_task["value"])
```

Comment:
This is totally nitpick on my side :) but the order of checks is different than in `test_Prize_Success`.